src/crimson/net/SocketConnection.cc

```diff
@@ -71,6 +71,7 @@ seastar::future<bool> SocketConnection::is_connected()
 
 seastar::future<> SocketConnection::send(MessageRef msg)
 {
+  logger().info("{} --> {} === {}", messenger, get_peer_addr(), *msg);
```

Comment:
info-level logs will be printed out by default, do you think it is the case for each message to send?

And connection itself can be printed to show its brief status including the peer address, usually looks like `("{} send: {}", *this, *msg)`

Reply:
> info-level logs will be printed out by default, do you think it is the case for each message to send?

probably we should use "trace" or "debug" instead ?

> And connection itself can be printed to show its brief status including the peer address

i tried to replicate what we have in classic osd. so ones familiar with the "debug ms" output is able to read this output immediately. i will experiment to see how "*this" looks like.

Reply:
> probably we should use "trace" or "debug" instead ?

I think we can put as "debug" firstly, and change to "trace" later if we feel it is too overwhelming.

> so developers who are used to the "debug ms" output will be able to read this output immediately.

We can improve the conn prefix if needed, maybe better in a separate PR/commit?

Reply:
i am not sure how to represent the 
```
{from} -> {to}
```
or
```
{to} <= {from}
```
using `SocketConnection::print()`. i think the one we have in classic is easier to read.

please see following examples:
```
ms - [osd.1(client) v1:192.168.2.8:6810/29763] --> v1:192.168.2.8:40841/0 === mon_subscribe({osdmap=15}) v3
```

```
INFO  2019-03-21 21:34:09,879 [shard 0] ms - [osd.1(hb_front) v1:192.168.2.8:6813/33709 >> v1:192.168.2.8:0/62610@37756] recv osd_ping(ping e30 stamp 2019-03-21 21:34:09.874605) v4
INFO  2019-03-21 21:34:09,879 [shard 0] ms - [osd.1(hb_front) v1:192.168.2.8:6813/33709 >> v1:192.168.2.8:0/62610@37756] send osd_ping(ping_reply e30 stamp 2019-03-21 21:34:09.874605) v4
```

and i am posting #27099 for easier review.